### ISSUE TITLE
Question regarding the source code for your paper, Controllable Machine Unlearning via Gradient Pivoting (CUP)

### ISSUE THREAD

**Issue by canshizzz** [hys3835/Dual-Cone-Gradient-Descent#3]
Hello, thank you so much for your highly constructive work!

I am a graduate student in Artificial Intelligence, currently researching machine unlearning. I have read your two papers, Controllable Machine Unlearning via Gradient Pivoting (CUP) and Dual Cone Gradient Descent for Training Physics-Informed Neural Networks (DCGD).

Would it be possible for you to share the source code for CUP to assist with my research?

I would greatly appreciate it!

**Comment by youngsikhwang** [comment]
Thank you for your interest in my research.

I will create a new GitHub repository and upload the code for CUP there soon.

If you have any questions, please feel free to contact me at: [hys3835@unist.ac.kr](mailto:hys3835@unist.ac.kr).
Also, I would be happy to discuss any potential research ideas or directions.

Thank you.

**Comment by youngsikhwang** [comment]
I have created a new GitHub repository for the project, "Controllable Machine Unlearning by Pivoting Gradient".

Thank you again for your interest in our research.
